tuf/src/client.rs

```diff
@@ -846,20 +900,41 @@ where
     pub async fn fetch_target_description(
         &mut self,
         target: &TargetPath,
+    ) -> Result<TargetDescription> {
+        self.fetch_target_description_with_start_time(target, &Utc::now())
+            .await
+    }
+
+    /// Fetch a target description from the remote repo and return it.
+    pub async fn fetch_target_description_with_start_time(
+        &mut self,
+        target: &TargetPath,
+        now: &DateTime<Utc>,
```

Comment:
nit: the `now` parameter is called `start_time` in some of the other fns

Reply:
ah you caught me. Fixed.